packages/libs/react-ui/.eslintrc.js

```diff
@@ -0,0 +1,13 @@
+// This is a workaround for https://github.com/eslint/eslint/issues/3458
+require('@rushstack/eslint-config/patch/modern-module-resolution');
+module.exports = {
+  extends: ['@kadena-dev/eslint-config/profile/react', 'plugin:storybook/recommended', 'plugin:storybook/recommended'],
+  parserOptions: {
+    tsconfigRootDir: __dirname
+  },
+  settings: {
+    react: {
+      version: 'detect'
+    }
+  }
```

Comment:
Is this not in the `@kadena-dev/eslint-config/profile/react`? Otherwise might be good to add it there.